starboard/tools/api_leak_detector/stub/qa/manifest

```diff
@@ -108,6 +108,7 @@ setlocale
 setvbuf
 snprintf
 sprintf
+sscanf
```

Comment:
I thought we were adding these symbols to a list in the api leak detector script itself, not the manifests?

Reply:
yeah, this should be either in the script or in the c99.md file.

Reply:
Moved to c99.md file